src/main/java/org/weasis/dicom/google/api/ui/SearchPanel.java

```diff
@@ -123,10 +129,22 @@ private void initSearchPanel() {
         JButton searchButton = new JButton("Search");
         searchButton.addActionListener((action) -> reloadTable());
```

Comment:
I am curious - say when someone has not searched for anything yet what will the UI look like ? I don't think we should display the navigation panel until someone has searched for something...

Reply:
originally i did it that way, but in some reason it was annoying for me when navigation shows up with data, but its subjective, if you think it's better to hide buttons i'll revert that part

Reply:
Interesting, do you mind uploading a screenshot with the current state and with hiding for not searched yet(if this is a big change then I would love to see the page with the current implementation and will let you know my thoughts)?

Reply:
![image](https://user-images.githubusercontent.com/878990/58716408-26a2b600-83f3-11e9-93c2-56948d0ddcd1.png)
![image](https://user-images.githubusercontent.com/878990/58716424-315d4b00-83f3-11e9-84f0-99837fe73b87.png)
if this is still confusing(for me as well) i could invite UX designer to this discussion
and i almost sure designer would say something like "whole this UI is rubbish"

Reply:
Thanks a lot for the screenshots :) I think I am leaning towards hiding the navigation pane in the beginning. Hehe I know - I did find it a little confusing in the beginning too - ideally I would think we should have just the search options and once someone searches for something we can display the results in the table but I think for now lets just hide the navigation panel until someone searches just because having that next button feels like someone might go try clicking that..

Reply:
ok

Reply:
done